backend/apps/ai/management/commands/ai_create_project_chunks.py

```
@@ -0,0 +1,225 @@
+    DEFAULT_LAST_REQUEST_OFFSET_SECONDS,
+    DELIMITER,
+    MIN_REQUEST_INTERVAL_SECONDS,
+)
+from apps.ai.models.chunk import Chunk
+from apps.owasp.models.project import Project
+
+
+class Command(BaseCommand):
+    help = "Create chunks for OWASP project data"
+
+    def add_arguments(self, parser):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)